### PR TITLE
Refresh homepage background image

### PR DESCRIPTION
**Pull request by kristinli19**
This commit just switches out our nice homepage image for an image of SF, based on Sophia's updated design! 

Look at this beautiful SF skyline:
image

### CODE REVIEW

File: pages/index.vue [hunk 1]
```diff
@@ -83,7 +83,7 @@ p {
 }
 
 .hero-image {
-  background-image: url('/olivia-bliss-HTnycx0h_kk-unsplash.jpg');
+  background-image: url('/hardik-pandya--Ey_0PMz900-unsplash.jpg');
```
Comment (jamesbaskerville) [inline pages/index.vue:86]: do you mind renaming this to something sane while you're here? e.g. `san-francisco-skyline.jpg`

File: pages/index.vue [hunk 2]
```diff
@@ -83,7 +83,7 @@ p {
 }
 
 .hero-image {
-  background-image: url('/olivia-bliss-HTnycx0h_kk-unsplash.jpg');
```
Comment (jamesbaskerville) [inline pages/index.vue:86]: Also can you remove the other image if we're gonna stop using it?

File: pages/index.vue (hunk 1 above)
Comment (kristinli19) [reply pages/index.vue:86]: yup makes sense - done!

File: pages/index.vue (hunk 2 above)
Comment (kristinli19) [reply pages/index.vue:86]: definitely :) done!

**Pull request merged**